tools/restatectl/src/commands/metadata/mod.rs

```diff
@@ -84,18 +62,17 @@ enum RemoteServiceType {
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct GenericMetadataValue {
-    // We assume that the concrete serialized type's encoded version field is called "version".
     version: Version,
 
     #[serde(flatten)]
-    data: serde_json::Map<String, serde_json::Value>,
+    fields: serde_json::Map<String, serde_json::Value>,
```

Comment:
Out of curiosity did we ever check that any flexbuffer serialized value can be deserialized as a map of json values? So far things seem to work. Maybe @pcholakov looked into it when introducing the `GenericMetadataValue`.